Packs/WebFileRepository/Integrations/WebFileRepository/WebFileRepository.py

```diff
@@ -4267,35 +4547,120 @@ def command_download_file(args: dict[str, str], settings: Settings) -> dict[str,
         raise DemistoException('A file path is required.')
 
     client = new_client(detect_service_ip_port(settings), settings)
-    resp = client._http_request('GET',
-                                params={
-                                    'q': 'download',
-                                    'path': path
-                                },
-                                raise_on_status=True,
-                                resp_type='response')
-
-    if not (filename := args.get('save_as')) and (content_disposition := resp.headers.get('Content-Disposition')):
-        cdp = email_parser.Parser().parsestr(f'Content-Disposition: {content_disposition}', headersonly=True)
+    resp = client._http_request(
+        'GET',
+        params={
+            'q': 'download',
+            'path': path
+        },
+        raise_on_status=True,
+        resp_type='response'
+    )
+    if (
+        not (filename := args.get('save_as'))
+
+        and (content_disposition := resp.headers.get('Content-Disposition'))
+    ):
+        cdp = email_parser.Parser().parsestr(
+            f'Content-Disposition: {content_disposition}',
+            headersonly=True
+        )
         filename = cdp.get_filename()
 
     return fileResult(filename or str(uuid.uuid4()), resp.content)
 
 
-def command_archive_zip(args: dict[str, str], settings: Settings) -> dict[str, Any]:
-    """ Archive all the files into a zip file
+def command_download_as_text(
+    args: dict[str, str],
+    settings: Settings
+) -> CommandResults:
+    """ Download a file from the repository, and set the data to the context
 
     :param args: The parameters which were given to the command.
     :param settings: The instance settings.
     """
+    if not (path := args.get('path')):
+        raise DemistoException('A file path is required.')
+
```

Comment:
setting the argument as required in the yml file enforces the user to insert a value to the parameter, so those lines can be removed

Reply:
@yucohen `git commit` fails by mypy when removing the check. The same check is also in the `command_download_file`. I think it would be better for the safety code. 
<img width="924" alt="image" src="https://github.com/demisto/content/assets/54964121/98db4e44-aa34-464c-92e6-3612cd219a79">

Reply:
cool. so I'm merging